### ISSUE TITLE
Scammers using https://validator.w3.org/#validate_by_input 

### ISSUE THREAD

**Issue by Jake-s16** [validator/validator#842]
Scammers are using https://validator.w3.org/#validate_by_input to "prove" something is wrong with a person's computer.

They just tried to use the warnings and errors that came up after I put my email address in. I had been playing along with them, no msconfig or eventvwr on linux so when they could not get these to work they sent me straight here.

Could you recognise email addresses and then pop in a single warning to stop scammers from deceiving the unwary? Something like "Warning: You entered an email address. Validating an email address suggests you may have been directed here by a scammer"

**Comment by sideshowbarker** [comment]
What’s your e-mail provider?

**Comment by Jake-s16** [comment]
Why is that? What difference would that make?

**Comment by sideshowbarker** [comment]
> Why is that? What difference would that make? The idea would be that ALL email addresses would get that warning so these scammers stop using validator.

Yeah, nevermind, you’re right — if the scammers are directing people to https://validator.w3.org/#validate_by_input, then it doesn’t matter who the e-mail provider is.

The thing is, the scammers previously directed people to type their e-mail addresses in the default https://validator.w3.org/#validate_by_uri form field. And in that case, what gets submitted is an actual URL, and there is a mitigation we can do to handle that URL case. In fact we already have such a mitigation in place.

Given the mitigation we put in place for the https://validator.w3.org/#validate_by_uri case, I think that’s why the scammers are now directing people to type their e-mail addresses into the https://validator.w3.org/#validate_by_input form field instead — because the way they were scamming people previously doesn’t work any more.

Anyway, it’s a bit harder to have the checker backend check for an e-mail address in the https://validator.w3.org/#validate_by_input input. But I have an idea how to do it, so I’ll give it a try.

By the way, thanks very much for taking time to report this.

**Comment by Jake-s16** [comment]
Excellent, thanks for getting on to it. 

It's got to the point where most calls on our landline are scammers. Gotta try and set up Lenny and have some laughs (Search youtube if you haven't heard of him).
